faststream/kafka/broker.py

```diff
@@ -93,6 +94,13 @@ def __init__(
             client_id (str): The client ID for the Kafka client.
             **kwargs: Additional keyword arguments.
         """
+        if protocol is None:
+            security = kwargs.get("security")
```

Comment:
Why are you getting it from kwargs? Add it to arguments directly, it is much explicit way (but don't forget to pass it in super)

Reply:
Fixed